model/fv_arrays.F90

```diff
@@ -244,7 +244,7 @@ module fv_arrays_mod
                              !< monotonicity constraint of Huynh, which is less diffusive
                              !< but more expensive than other monotonic constraints. For hydrostatic simulation, 8
                              !< (the L04 monotonicity constraint) or 10 are recommended; for
-                             !< nonhydrostatic simulation, the completely unlimited (“linear”
```

Comment:
Were these quote marks removed because of a problem with some compilers?

Reply:
Not compilers. It is my editor (vim) that cannot render those characters properly. Anyway, I have restored the file to its original format. Sorry for the trouble.

I have tested the implementation with the NSSL 2-moment MP scheme and the Thompson scheme. Unfortunately, it produces zero output for the GFDL MP scheme. John Henderson is looking at the problem for a possible reason.

Reply:
OK, thanks. 

The GFDL MP doesn't have second moments which may confuse the diagnostic.